RFCs/004-In-Assembly-Parallel-Execution.md

```diff
@@ -66,6 +66,13 @@ Test lifecyle method semantics
 - ClassInitialize/Cleanup shall be run only once per class (irrespective of parallel or not).
 - TestInitialize/Cleanup shall be run only once per method.
 
+## Conditioning in-assembly parallel execution - composition rules
+In-assembly parallel execution can be conditioned using the following means:
+1. as annotations in source code (as described in this document).
```

Comment:
Please provide example here for all rules with precedence.

Reply:
Just added a sample.